common/containers/Tabs/GenerateWallet/components/EnterPassword.tsx

```diff
@@ -1,41 +1,37 @@
 import { GenerateNewWalletAction } from 'actions/generateWallet';
 import React, { Component } from 'react';
 import { Link } from 'react-router-dom';
-import { Field, reduxForm } from 'redux-form';
 import translate from 'translations';
 import './EnterPassword.scss';
 import PasswordInput from './PasswordInput';
 import Template from './Template';
+
 // VALIDATORS
-const minLength = min => value => {
-  return value && value.length < min
-    ? `Must be ${min} characters or more`
```

Comment:
AFAIK, these validation messages are not actually displayed to the user and thus were removed.